lib/run/src/Obelisk/Run.hs

```diff
@@ -191,26 +192,26 @@ runWidget toRun configs validFullEncoder = do
       -- specified in the route.
       port = fromMaybe 80 $ (_runApp_forceFrontendPort toRun)
                         <|> (fmap fromIntegral $ uri ^? uriAuthority . _Right . authPort . _Just)
-      -- This is the *actual* URI on which the frontend is served, i.e.
-      -- the URI from the route configuration but, possibly, with the
-      -- port we picked above. We need to compute this for two reasons:
-      --
-      --   1. The log. Self-explanatory.
-      --   2. JSaddle needs to know where the frontend is served.
-      actualUri = uri & uriAuthority . _Right . authPort ?~ fromInteger (fromIntegral port)
 
       -- This is the server that will handle the backend requests. We
       -- support shuttling them off to any host:port pair.
       redirectHost = _runApp_backendHost toRun
       redirectPort = _runApp_backendPort toRun
 
       beforeMainLoop = do
-        putStrLn $ "Frontend running on " ++ T.unpack (URI.render actualUri)
+        putStrLn $ "Frontend running on http://localhost:" ++ show port ++ "/"
```

Comment:
So before, this was logging the public-facing route, and now it's logging where `ob run` is listening on the local machine?